exercises/oop/geometric_shapes/deivio-solution/Square.js

```diff
@@ -0,0 +1,23 @@
+export default
+
+class Square{
+    constructor(Side) {
```

Comment:
Side written capitalized. Bad practice as capitalized names usually indicate types and classes, not variables.